client/src/components/Auth/Login.tsx

```diff
@@ -136,6 +146,14 @@ function Login() {
             {localize(getLoginError(error))}
           </div>
         )}
+        {customError && (
```

Comment:
Just needs merge conflict resolved and I think this is good to go!

Reply:
Aight I'm okay with this!